Tribler/Core/DownloadConfig.py

```diff
@@ -193,8 +194,8 @@ def load(filename):
         dlconfig = CallbackConfigParser()
         try:
             dlconfig.read_file(filename)
-        except:
-            raise IOError, "Failed to open download config file"
+        except (ParsingError, IOError):
+            raise IOError, "Failed to open download config file: %s" % filename
```

Comment:
Also, if I'm not mistaken, the exact content of the exception (the cause) is not printed? So, if the `ParsingError` happens, I only see `Failed to open download config file: ...` and not what went wrong during parsing.